README.md

```diff
@@ -0,0 +1,18 @@
+freemex
+--------
+                                 Building from source
```

Comment:
Please see how markdown works. In github you can see a preview of the markdown file.
This is not proper syntax bringing int to the right by appending some spaces.

use single # for h1
use double ## for h2 .. like that.

See:
https://daringfireball.net/projects/markdown/syntax